Missing boost_system library

Building on Ubuntu 19.10 via `make` results in

```
clang++ koradcontrol.cpp -lboost_program_options -pthread -Wall -o koradcontrol --static
/usr/bin/ld: /tmp/koradcontrol-fc1316.o: in function `boost::asio::error::get_system_category()':
koradcontrol.cpp:(.text._ZN5boost4asio5error19get_system_categoryEv[_ZN5boost4asio5error19get_system_categoryEv]+0x5): undefined reference to `boost::system::system_category()'
/usr/bin/ld: /tmp/koradcontrol-fc1316.o: in function `boost::system::error_category::std_category::equivalent(int, std::error_condition const&) const':
koradcontrol.cpp:(.text._ZNK5boost6system14error_category12std_category10equivalentEiRKSt15error_condition[_ZNK5boost6system14error_category12std_category10equivalentEiRKSt15error_condition]+0xc1): undefined reference to `boost::system::generic_category()'
/usr/bin/ld: koradcontrol.cpp:(.text._ZNK5boost6system14error_category12std_category10equivalentEiRKSt15error_condition[_ZNK5boost6system14error_category12std_category10equivalentEiRKSt15error_condition]+0xf9): undefined reference to `boost::system::generic_category()'
/usr/bin/ld: /tmp/koradcontrol-fc1316.o: in function `boost::system::error_category::std_category::equivalent(std::error_code const&, int) const':
koradcontrol.cpp:(.text._ZNK5boost6system14error_category12std_category10equivalentERKSt10error_codei[_ZNK5boost6system14error_category12std_category10equivalentERKSt10error_codei]+0xc0): undefined reference to `boost::system::generic_category()'
/usr/bin/ld: koradcontrol.cpp:(.text._ZNK5boost6system14error_category12std_category10equivalentERKSt10error_codei[_ZNK5boost6system14error_category12std_category10equivalentERKSt10error_codei]+0xf2): undefined reference to `boost::system::generic_category()'
/usr/bin/ld: koradcontrol.cpp:(.text._ZNK5boost6system14error_category12std_category10equivalentERKSt10error_codei[_ZNK5boost6system14error_category12std_category10equivalentERKSt10error_codei]+0x218): undefined reference to `boost::system::generic_category()'
/usr/bin/ld: /tmp/koradcontrol-fc1316.o: in function `boost::system::error_code::error_code()':
koradcontrol.cpp:(.text._ZN5boost6system10error_codeC2Ev[_ZN5boost6system10error_codeC2Ev]+0x1b): undefined reference to `boost::system::system_category()'
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [Makefile:2: koradcontrol] Error 1
```

Linker seems to require `boost_system` library (see e.g. https://stackoverflow.com/questions/9723793/undefined-reference-to-boostsystemsystem-category-when-compiling)

Solved by adding `-lboost_system` in Makefile

--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,5 @@
 koradcontrol : koradcontrol.cpp
-	clang++ koradcontrol.cpp -lboost_program_options -pthread -Wall -o koradcontrol --static
+	clang++ koradcontrol.cpp -lboost_program_options -lboost_system -pthread -Wall -o koradcontrol --static
 
 clean:
 	rm koradcontrol

